python/optify/tests/test_provider.py

```diff
@@ -20,4 +20,4 @@ def test_features():
     except:
         # Can't get pyo3_runtime.PanicException because can't import it and catching `Exception` doesn't work either.
```

Comment:
Drive-by comment: I think `BaseException` would work

Reply:
Thanks that works. I found some history on it here: https://github.com/PyO3/pyo3/issues/3519